src/pages/event-schema.ts

```diff
@@ -14,8 +14,20 @@ export const eventDateSchema = Joi.date()
     "date.max": `Date must be before ${maxDate.toISOString().split("T")[0]}`,
   });
 
+export const endDateSchema = Joi.date()
+  .iso()
+  .allow("")
+  .min(Joi.ref("startDate"))
+  .max(maxDate)
+  .messages({
+    "date.format": `Date must be a real date`,
+    "date.min": `End date must be after start date`,
```

Comment:
can we also reword these error messages. End date can also be the same as the start date